src/test/lib/storage/segment_iterators_test.cpp

```diff
@@ -194,7 +194,7 @@ TEST_P(SegmentIteratorsTest, LegacyRandomIteratorCompatible) {
 }
 
 template <typename T>
-bool operator<(const AbstractSegmentPosition<T>&, const AbstractSegmentPosition<T>&) {
+bool operator<(const AbstractSegmentPosition<T>& /*lhs*/, const AbstractSegmentPosition<T>& /*rhs*/) {
```

Comment:
`unused`? Again, especially in this case, I see the benefit of using more descriptive comments.